README.md

```diff
@@ -1,2 +1,28 @@
-# ugics-2018-android-basics-wiki
-Wiki for Udacity-Google-India-Challenge-Scholarship-2018 :mortar_board: Android-Basics-Course :ledger:
+# :mortar_board: :mortar_board: Wiki - Android Basics Track :mortar_board: :mortar_board:
+## A Udacity Google India Challenge Scholarship 2018 student initiative
+A wiki :ledger: for Udacity Google India Challenge Scholarship 2018 Android Basics Track.
+<p align="center">
+  <img width="300" height="300" src="https://imgur.com/FrGft2b.png">
+</p>
+
+### General
+* [Archived Slack Messages - Slark](https://slarck.com/9z1N5S/?k=Xkqwg9YczIT9xRPqAnMvPj7brNC3F8qFw5dOyS8M)
+* [Forums](https://discussions.udacity.com/c/standalone-courses/ud834-in/)
+* New Workspace Link - _to be updated after May 29th_
+* [LinkedIn Profiles](https://docs.google.com/spreadsheets/d/1JqGTMa6AQJTjHZT1w7tU76-9h6K3vdxAj1A5yNlk3mg) _Not recommended as public documents can be abused outsiders_
```

Comment:
Will it be fine if we keep this in README?

Reply:
I'm only concerned about some outsider changing it and I don't know a solution. What should we do?

Reply:
I guess I misread that line, My bad.

Reply:
We can try talking about this with owner of spreadsheet.

Reply:
Sure. Will do.